@test(groups=['rgm'], depends_on=[testImportAirlines])
def testGoodParameters(self):
    dl_test_parameters = {'classification': True, 'response_column': 'CAPSULE', 'hidden': '[10, 20, 10]', }
    parameters_validation = self.a_node.validate_model_parameters(algo='deeplearning', training_frame=self.prostate_key, parameters=dl_test_parameters, timeoutSecs=self.timeoutSecs)
    assert ('validation_error_count' in parameters_validation), 'Failed to find validation_error_count in good-parameters parameters validation result.'
    h2o.H2O.verboseprint('Bad params validation messages: ', repr(parameters_validation))
    assert (0 == parameters_validation['validation_error_count']), '0 == validation_error_count in good-parameters parameters validation result.'
